.github/CONTRIBUTING.md

```diff
@@ -117,6 +117,19 @@ To link your Codecademy user profile to GitHub:
 - Typos/bugs
 - Plagiarism
 
+## How to keep a PR branch upto date
+
+To keep your PR branch up to date navigate to the branch on your fork. Then press `Fetch upstream` and `Fetch and merge`.
+
+![Image of where the `Fetch upstream` button is](https://raw.githubusercontent.com/Codecademy/docs/main/media/branch-up-tp-date-1.png)
+
+![Image of the `Fetch and merge` button is](https://raw.githubusercontent.com/Codecademy/docs/main/media/branch-up-to-date-2.png)
+
+If you want to use the command line with git run the following commands:
+`git remote add upstream https://github.com/Codecademy/docs.git`
+`git fetch upstream`
+`git rebase upstream master`
```

Comment:
A few thoughts here:

* Let's change `master` to `main` since our own Docs repo uses `main` as the name of the base branch.
* Let's also turn this into a multi-line \```bash code snippet like this:

```git
git remote add upstream https://github.com/Codecademy/docs.git
git fetch upstream
git rebase upstream main
```